modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -525,6 +557,12 @@ export default function EditNotificationTemplate({
 
 						<div className="col-lg-6 lfr__notification-template-card">
 							<Card title={Liferay.Language.get('settings')}>
+								<Text as="span" color="secondary">
+									{Liferay.Language.get(
+										'use-terms-to-populate-fields-dynamically'
+									)}
+								</Text>
+
```

Comment:
Hey @naraandrade, should this be restricted by the feature flag?

Reply:
Oh, yes. This already fits the actual behavior on master, but I think it would be better to hide it for now.